app/assets/javascripts/controllers/middleware_jdbc_driver/middleware_jdbc_driver_controller.js

```diff
@@ -29,10 +29,10 @@ function MwAddJdbcDriverController($scope, $http, miqService) {
     })
       .then(
         function(result) { // success
-          miqService.miqFlash(result.data.status, result.data.msg);
+          miqService.replacePartials(result.data);
         },
         function() { // error
-          miqService.miqFlash('error', 'Unable to add JDBC Driver "' + data.driverName + '" on this server.');
+          miqService.replacePartials(result.data);
```

Comment:
The think with the flash  function   `miqService.miqFlash` is that doesn't allow me to show more than one message,  I saw that this helper is used in other parts of the code, I just use it.  If there is a better way for do this, I can do the change.